src/layouts/sidebar/index.tsx

```diff
@@ -100,13 +110,45 @@ export default function Sidebar() {
       icon: <CategoryIcon fontSize='large' />,
       path: ROUTE.CATEGORY,
     },
+  ];
+
+  const superAdminAccessSidebar = [
+    ...adminAccessSidebar,
     {
-      label: 'Ticket',
-      icon: <ConfirmationNumberIcon fontSize='large' />,
-      path: ROUTE.TICKET,
+      label: 'Organization',
+      icon: <ApartmentIcon fontSize='large' />,
+      path: ROUTE.ORGANIZATION,
     },
   ];
 
+  const departmentHeadAccessSidebar = [
+    ...employeeAccessSidebar,
+    {
+      label: 'Department',
+      icon: <DomainIcon fontSize='large' />,
+      path: ROUTE.DEPARTMENT,
+    },
+  ];
+
```

Comment:
We can actually move all this sidebar related code to a separate file. Lets say getSidebarConfig.tsx
And move all this code there into a function which returns finally the sidebarConfig. We just need to pass the role to that function

Reply:
okay